playbooks/group_vars/all

```diff
@@ -1,3 +1,3 @@
 env: stage
-secure_dir: '../../configuration-secure/ansible'
+secure_dir: '../../../configuration-secure/ansible'
```

Comment:
TODO: We can leave this unchanged. Separate PR will add this explicitly for each env.